### ISSUE TITLE
Not requesting status update from car

### ISSUE THREAD

**Issue by nilstusing** [yp87/leaf2mqtt#65]
**Describe the bug**
Leaf2mqtt seems to fail requesting status update from the car. It is very wierd. If i request an update for battery status in the Nissan App, it will also update in mqtt a few moments later. But if i just leave leaf2mqqt, it never updates. I can see that it publishes new messages in mqtt broker, but the data is the same until i manually request an update in Nissan app or My Leaf. 

Any tips?

**LEAF_TYPE**
newerThanMay2019

**Expected behavior**
Automatically getting status update from car.

**Comment by antonhagg** [comment]
Have you found out any more about this issue?

**Comment by nilstusing** [comment]
I have not. Kinda gave up on this. Do you have similar issues?

**Comment by antonhagg** [comment]
Yes, I have a similar issue, #71 and probably similar to #67 aswell. @yp87 is refering to @kamiKAC and he/she has done a pull request, so I guess @yp87 need to merge to change (hopefully this fixes the issue).

**Comment by antonhagg** [comment]
In the meantime, I have a workaround, and that is requesting an update from my mqtt-client with topic:leaf/command/battery and payload:update

**Comment by yp87** [comment]
Hello, I will not merge anything since I cannot test the changes anymore. As said in the project description this project is not active anymore. Please try the version from Kamikac: Docker tag is kamikac/leaf2mqtt
